bitcoin/src/blockdata/script.rs

```diff
@@ -559,17 +579,7 @@ impl Script {
     /// Checks whether a script pubkey is a P2PK output.
```

Comment:
Suggested additional paragraph:

> You can obtain the public key, if it's valid, by calling ``[`p2pk_public_key()`](Self::p2pk_public_key)``.

Reply:
Unrelated question @Kixunil, it seems this form of rustdoc is favouring the rendered docs at the expense of readability in an editor, wouldn't plain ``[`Self::p2pk_public_key()`]`` be better?

Reply:
:+1:

Reply:
@tcharding I always prefer rendered docs because it's much easier to navigate through them. But it it annoys anyone putting the link at the bottom is fine. :)